libraries/tests/utest/serial_asynch/serial_asynch.cpp

```diff
@@ -199,7 +203,13 @@ TEST(Serial_Asynchronous, rx_parity_error)
     while ((!tx_complete) || (!rx_complete));
 
     CHECK_EQUAL(SERIAL_EVENT_TX_COMPLETE, tx_event_flag);
+#if defined(TARGET_B96B_F446VE)
```

Comment:
Please remove these #if def from this test. It's fine if it fails for now, it can be fixed. But it should be noted somewhere this different behavior for now.